plugins/MantisCoreFormatting/files/markdown.css

```diff
@@ -1,3 +1,24 @@
+/*
+ * Rules taken from bootstrap.
+ *
+ * The right way is creating boostrap css from scss sources instead
+ * of override the rules here.
+ *
+ * @todo fix this directly in bootstrap and delete this rules.
+ */
+code {
+    color: #333;
+    white-space: pre;
+    padding: 0.2rem .4rem 0.1rem .4rem;
+    margin: 0;
+    font-size: 90%;
+    background-color: #f5f5f5;
+    border-radius: 4px;
+}
```

Comment:
Is it necessary to redefine the whole rule ? Wouldn't it be enough to only override the necessary properties (I would guess, *color* and *background-color*.

Reply:
Not very precise :-( I can do better.